mlflow/R/mlflow/R/python.R

```diff
@@ -12,7 +12,7 @@ get_python_bin <- function() {
     stop(paste("MLflow not configured, please run install_mlflow() or ",
                "set MLFLOW_PYTHON_BIN and MLFLOW_BIN environment variables.", sep = ""))
   }
-  mlflow_env$python
+  file.path(dirname(mlflow_env$python), "python.exe")
```

Comment:
Are we sure this line is correct?

Reply:
Does `python.exe` exist in Linux?

Reply:
Or do we use this function only in Windows?

Reply:
@WeichenXu123 @serena-ruan

Reply:
Sorry I think it's in windows, I'll fix it